cogito/putter_private_test.go

```diff
@@ -345,13 +345,19 @@ func TestConcourseBuildURL(t *testing.T) {
 			name: "instanced vars 1",
 			env: testhelp.MergeStructs(baseEnv,
 				Environment{BuildPipelineInstanceVars: `{"branch":"stable"}`}),
-			want: "https://ci.example.com/teams/devs/pipelines/magritte/jobs/paint/builds/42?vars=%7B%22branch%22%3A%22stable%22%7D",
+			want: "https://ci.example.com/teams/devs/pipelines/magritte/jobs/paint/builds/42?vars.branch=%22stable%22",
```

Comment:
@marco-m-pix4d should we start using go-vcr pkg in these tests?

I wonder if we could have catch this sooner if we used it already.

Reply:
> should we start using go-vcr pkg in these tests?

Probably.

> I wonder if we could have catch'ed this sooner if we used it already.

This is the part I don't understand. I think that I tested against https://github.com/marco-m/concourse-in-a-box, so I don't understand why this now fails. True in any case that if we had a go-vcr trace, we could answer this question...